iceoryx_hoofs/platform/win/include/iceoryx_hoofs/platform/platform_settings.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+#ifndef IOX_HOOFS_LINUX_PLATFORM_PLATFORM_SETTINGS_HPP
+#define IOX_HOOFS_LINUX_PLATFORM_PLATFORM_SETTINGS_HPP
+
+#include <cstdint>
+
+namespace iox
+{
+namespace posix
+{
+class NamedPipe;
+}
+} // namespace iox
+
+// windows has two possible path separators
+constexpr const char IOX_PATH_SEPARATORS[] = "\\/";
+constexpr uint64_t IOX_UDS_SOCKET_MAX_MESSAGE_SIZE = 4096;
+constexpr char IOX_UDS_SOCKET_PATH_PREFIX[] = "";
+using IoxIpcChannelType = iox::posix::NamedPipe;
+
+// just increase this number to increase the maximum shared memory size supported
+// by windows
+constexpr uint64_t IOX_MAXIMUM_SUPPORTED_SHM_SIZE = 1024 * 1024 * 1024;
```

Comment:
This is quite low and only used on Windows. Any reason to limit it to only one GB? Could the platform specific setting be added to a `win32` namespace to prevent accidental usage outside of the platform? ... oh, the settings are not put into a namespace at all. What's the reason for this?

Reply:
You are right on the other hand, what would be a right number? 4? 8?
I will increase this to 8GB.

Reply:
The right amount would be π